core-bundle/src/Resources/contao/widgets/RootPageDependentSelect.php

```diff
@@ -29,7 +29,7 @@ public function generate(): string
 		$rootPages = $framework->getAdapter(PageModel::class)->findByType('root', array('order' => 'sorting'));
 		$wizard = StringUtil::deserialize($this->wizard);
 
-		$this->blankOptionLabel = $translator->trans(sprintf('tl_module.%sBlankOptionLabel', $this->name), array(), 'contao_module');
+		$this->blankOptionLabel = $translator->trans(sprintf('tl_module.%sBlankOptionLabel', $this->name), array(), 'contao_tl_module');
```

Comment:
This cannot have worked, either. We might have to backport the fix.

Reply:
Yes, that should go to 4.13 and then get merged upstream :)

Reply:
I can provide a fix PR for 4.13 if you want :)

Reply:
This works as intended in my 4.13 (`dump($this->blankOptionLabel);`) on line 34:

<img width="252" alt="Bildschirmfoto 2022-04-20 um 07 06 55" src="https://user-images.githubusercontent.com/754921/164154522-a5ed84b1-9ab7-43f9-b286-33afb86d9d9b.png">

Reply:
I wonder why, because `contao_module` will try to load a `module.xlf` file which definitely does not exist.

Reply:
Is there somewhere a magic fallback if no translation is found on the given domain?

Reply:
No, our translator simply reads from `$GLOBALS['TL_LANG']` if the domain begins with `contao_`. And apparently `$GLOBALS['TL_LANG']['tl_module']` exists already, therefore we did not notice this. Fixed in #4521 now.